vipps-ecom-api.md

```diff
@@ -2214,44 +2214,44 @@ to retrieve all the information about the payment.
 
 | Error group | Error Code | Error Message (and some explaining text for some errors) |
 | ----------- | ---------- | -------------------------------------------------------- |
-| Payment     | 41         | The user does not have a valid card |
-| Payment     | 42         | Refused by issuer bank |
-| Payment     | 43         | Refused by issuer bank because of invalid a amount |
-| Payment     | 44         | Refused by issuer because of expired card |
-| Payment     | 45         | Reservation failed. Either because the user cancelled, did not act, or for some unknown reason. |
-| Payment     | 51         | Cannot cancel an already captured order |
-| Payment     | 52         | Cancellation failed |
-| Payment     | 53         | Cannot cancel an order that is not reserved |
-| Payment     | 61         | Captured amount exceeds the reserved amount |
-| Payment     | 62         | The amount you tried to capture is not reserved |
-| Payment     | 63         | Capture failed for some unknown reason, please use /details endpoint to know the exact status |
-| Payment     | 71         | Cannot refund more than captured amount |
-| Payment     | 72         | Cannot refund a reserved order (only captured orders), please use the /cancel endpoint |
-| Payment     | 73         | Cannot refund a cancelled order |
-| Payment     | 93         | Captured amount must be the same in an Idempotent retry |
-| Payment     | 95         | Payments can only be refunded up to 365 days after reservation |
-| Payment     | 96         | Payments can only be captured up to 180 days after reservation |
+| Payment     | 41         | The user does not have a valid card. |
+| Payment     | 42         | Refused by issuer bank. Vipps can not provide more details. |
+| Payment     | 43         | Refused by issuer bank because of invalid a amount. Vipps can not provide more details. |
+| Payment     | 44         | Refused by issuer because of expired card. Vipps can not provide more details. |
+| Payment     | 45         | Reservation failed. Either because the user cancelled, the user did not act, or for some unknown reason. |
```

Comment:
"or for some unknown reason."

Should just be removed? I don't think it's applicable anymore. Rather replace with "generic error given in the case of payment not being acted upon by Customer".